doc/source/whatsnew/v0.24.0.txt

```diff
@@ -121,6 +121,7 @@ Other Enhancements
 - :class:`IntervalIndex` has gained the :meth:`~IntervalIndex.set_closed` method to change the existing ``closed`` value (:issue:`21670`)
 - :func:`~DataFrame.to_csv` and :func:`~DataFrame.to_json` now support ``compression='infer'`` to infer compression based on filename (:issue:`15008`)
 - :func:`to_timedelta` now supports iso-formated timedelta strings (:issue:`21877`)
+- :func:`read_json` now parse timedelta with `orient='table'` (:issue:`21140`)
```

Comment:
I might have missed this on previous comment but there's a small typo here. `parse` -> `parses`

Reply:
@WillAyd I've pushed fix, but now there is a merge conflict. How should I deal with it on github?

Reply:
Better to deal locally and re-push. Assuming you have things set up as mentioned in the pandas contributing guide do this on your local branch

```sh
git fetch upstream
git merge upstream/master
```

You'll probably get a merge conflict there, so fix that up and do:

```sh
git merge --continue
```

Everything should be resolved then so re-push after that